documentation/modules/auxiliary/dos/http/tautulli_shutdown_exec.md

```diff
@@ -0,0 +1,30 @@
+# Tautulli 2.1.9 - Shutdown Denial of Service
+
+## Overview
+Tautulli versions 2.1.9 and prior are vulnerable to denial of service via the /shutdown URL.
```

Comment:
Any other details? I believe I saw something about either a CSRF bypass, or that it was one of the few URLs not having CSRF protection.  Wrap /shutdown in ticks/tilda (``) so it should be: `/shutdown`